src/ai/susi/DAO.java

```diff
@@ -110,6 +110,7 @@
     public static JsonTray group;
     public static JsonTray skillRating;
     public static JsonTray fiveStarSkillRating;
+    public static JsonTray sevenDaysSkillUsage;
```

Comment:
I think instead of calling it `sevenDays...` explicitly, maybe it's a good idea to just call it `SkillUsage` where the default parameter retrieves 7 days. Otherwise, the user should be allowed to query the duration for the skill usage. That'd make this generic and nicer.

Reply:
Hmm but I thought  of keeping the stats of just last 7 days, otherwise a lot of data will be generated. We can extend it later, but my suggestion to first test with limited data on the production server, so that if any bug is encountered then its easy to fix it. Still I would like to have more views on this.

Reply:
Please make it generic so we can scale as it fits.
If we do any changes in the time frame then the name of the API is inappropriate. We would never go through all client applications and change this and we would be stuck in inappropriate naming.

Reply:
Ok, got your point. Sending a commit.